test/enigma_test.rb

```diff
@@ -12,12 +12,65 @@ def test_it_exists
     assert_instance_of Enigma, @enigma
   end
 
+  def test_it_has_attributes
+    expected = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m",
+       "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z", " "]
+    assert_equal expected, @enigma.alphabet
+  end
```

Comment:
Refactored the `Enigma` test suite to more accurately test the functionality of the class, as well as adding tests for new methods.